src/configLoader/getContent.js

```diff
@@ -61,3 +61,25 @@ function getConfigContent (configPath, baseDirectory) {
     const content = readConfigContent(resolvedPath);
     return getNormalizedConfig(configBasename, content);
 };
+
+/**
+ * Read proper content from config file.
+ * If the chartset of the config file is not utf-8, one error will be thrown.
+ * @param {String} configPath 
+ * @return {String}
+ */
+function readConfigFileContent (configPath) {
+
+    if (!configPath) {
+        throw new Error("config file path couldn't be empty");
+    }
+
+    let rawBufContent = fs.readFileSync(configPath);
+    let tmpBuf = Buffer.from(rawBufContent.toString("UTF-8"), "UTF-8");
+
+    if (!rawBufContent.equals(tmpBuf)) {
```

Comment:
Instead of trying to convert back and forth, I think it's better to use something like this: https://github.com/hcodes/isutf8